src/tickit/core/management/schedulers/master.py

```diff
@@ -60,7 +60,8 @@ def add_wakeup(self, component: ComponentID, when: SimTime) -> None:
         self.new_wakeup.set()
 
     async def run_forever(self) -> None:
-        """Perform an intial tick then continiously schedule ticks according to wakeups.
+        """Perform an initial tick then continuously schedule ticks according to
+        wakeups.
```

Comment:
I had to split this over two lines for flake8 but now pydocstyle has a warning in vs code.
Is this the correct way to format this?

Reply:
Does indenting the line remove the warning?

Reply:
From the [Google style guide](https://google.github.io/styleguide/pyguide.html) which Pydocstyle is configured to follow:
> A docstring should be organized as a summary line (one physical line not exceeding 80 characters) terminated by a period, question mark, or exclamation point. When writing more (encouraged), this must be followed by a blank line, followed by the rest of the docstring starting at the same cursor position as the first quote of the first line. There are more formatting guidelines for docstrings below.

Basically, you gotta remove some words